cypress.json

```diff
@@ -1,4 +1,3 @@
 {
-  "baseUrl": "http://localhost:3000",
-  "chromeWebSecurity": false
```

Comment:
Removing `chromeWebSecurity` reenables it as the default value is `true`. I thought this was an issue with Cypress in a CI/CD environment as others had reported similar issues about CORS, but in this case, it was because the error page rendered in plain text was I believe making it think it was a different domain other than `localhost`.

Long story short, this should be enabled normally.